File: app/src/main/java/com/tunasteak/activity/TwoWayBindingActivity.java
package com.tunasteak.activity;

import android.view.View;
import com.tunasteak.bean.SelectBean;
import com.tunasteak.demo.R;
import com.tunasteak.demo.databinding.ActivityTwoWayBindingBinding;
import com.tunasteak.model.TBindingModel;

/**
 * @author TunaSashimi
 * @date 2020-04-12 20:30
 * @Copyright 2020 TunaSashimi. All rights reserved.
 * @Description
 */
public class TwoWayBindingActivity extends TBindingActivity<ActivityTwoWayBindingBinding> {
    @Override
    protected int getLayoutId() {
        return R.layout.activity_two_way_binding;
    }

    @Override
    protected TBindingModel getModel() {
        return null;
    }

    @Override
    protected Object getData() {
        return null;
    }

    @Override
    public void onInit() {
        SelectBean selectBean = new SelectBean();
        binding.setBean(selectBean);
        selectBean.select.set(true);
    }

    @Override
    public void onClick(View v) {

    }
}

File: app/src/main/java/com/tunasteak/activity/TwoWayBindingActivity.kt
package com.tunasteak.activity

import android.view.View
import com.tunasteak.activity.TBindingActivity
import com.tunasteak.demo.R
import com.tunasteak.model.TBindingModel
import com.tunasteak.bean.SelectBean
import com.tunasteak.demo.databinding.ActivityTwoWayBindingBinding

/**
 * @author TunaSashimi
 * @date 2020-04-12 20:30
 * @Copyright 2020 TunaSashimi. All rights reserved.
 * @Description
 */
class TwoWayBindingActivity : TBindingActivity<ActivityTwoWayBindingBinding?>() {
    override fun getLayoutId(): Int {
        return R.layout.activity_two_way_binding
    }

    override fun getModel(): TBindingModel<*>? {
        return null
    }

    override fun getData(): Any? {
        return null
    }

    override fun onInit() {
        val selectBean = SelectBean()
        binding!!.bean = selectBean
        selectBean.select.set(true)
    }

    override fun onClick(v: View) {}
}